app/assets/javascripts/discourse/app/components/modal/reject-reason-reviewable.js

```diff
@@ -0,0 +1,17 @@
+import Component from "@glimmer/component";
+import { action } from "@ember/object";
+
+export default class RejectReasonReviewableModal extends Component {
```

Comment:
Is it possible to make these new modals into `.gjs` files? Or maybe modern-js is not doing this for now, and will do it all at once later?

Reply:
If it happens naturally, then that's fine, but I don't really want GJS conversions to block us on the last few modal updates. Hopefully at some point in future there will be a codemod which lets us do gjs conversions in bulk 🤞

Reply:
Yep makes sense, I wasn't sure :+1: